crates/core/tedge/src/cli/http/cli.rs

```diff
@@ -0,0 +1,280 @@
+use crate::cli::http::command::HttpAction;
+use crate::cli::http::command::HttpCommand;
+use crate::command::BuildCommand;
+use crate::command::BuildContext;
+use crate::command::Command;
+use crate::ConfigError;
+use anyhow::anyhow;
+use anyhow::Error;
+use camino::Utf8PathBuf;
+use certificate::CloudRootCerts;
+use clap::Args;
+use reqwest::blocking;
+use reqwest::Identity;
+use std::fs::File;
+use tedge_config::OptionalConfig;
+use tedge_config::ProfileName;
+
+#[derive(clap::Subcommand, Debug)]
+pub enum TEdgeHttpCli {
+    /// POST content to thin-edge local HTTP servers
+    ///
+    /// Examples:
+    ///   # Create a new Cumulocity Managed Object via the proxy service
```

Comment:
Would be nice to include one more example of the `/tedge/entity-store` service endpoint as well, as the other examples already covers the `/tedge/file-transfer` and `/c8y` endpoints.

Reply:
Added: https://github.com/thin-edge/thin-edge.io/pull/3357/commits/eac4dd0d9b92cc0fc2dc0863806db68845d80453